diff -ruN gem-0.92-1-old/src/Make.config.in gem-0.92-1/src/Make.config.in
--- gem-0.92-1-old/src/Make.config.in	2009-09-22 22:47:27.000000000 +0200
+++ gem-0.92-1/src/Make.config.in	2009-09-22 22:48:17.000000000 +0200
@@ -3,11 +3,10 @@
 
 CXX ?= @CXX@
 CXXFLAGS ?= @CXXFLAGS@
-LDFLAGS  ?= @LDFLAGS@
 LIBS     ?= @LIBS@
 
 GEM_CXXFLAGS= $(CXXFLAGS) @X_CFLAGS@ @PKG_CFLAGS@ -I.. @INCLUDES@ @DEFINES@
-GEM_LDFLAGS = $(LDFLAGS) @X_LIBS@
+GEM_LDFLAGS = $(LDFLAGS) @X_LIBS@ @LDFLAGS@
 GEM_LIBS = $(LIBS) @PKG_LIBS@
 
 GEM_ARCH_CXXFLAGS = @ARCH_FLAGS@
